config/testdata/telegraf-agent.toml

```diff
@@ -176,14 +176,6 @@
   # If no servers are specified, then 127.0.0.1 is used as the host and 4020 as the port.
   servers = ["127.0.0.1:4021"]
 
-# Read metrics from local Lustre service on OST, MDS
-[[inputs.lustre2]]
-  # An array of /proc globs to search for Lustre stats
-  # If not specified, the default will work on Lustre 2.5.x
-  #
-  # ost_procfiles = ["/proc/fs/lustre/obdfilter/*/stats", "/proc/fs/lustre/osd-ldiskfs/*/stats"]
-  # mds_procfiles = ["/proc/fs/lustre/mdt/*/md_stats"]
-
```

Comment:
this should not be removed

Reply:
I noticed this too but I think it's ok to remove because the config tests that use this file don't refer to lustre2 plugin.  The plugin isn't being built on windows so if it is left in, the tests would fail on windows.